### PR TITLE
Cleanup nuodb-operator

### PR DESCRIPTION
**Pull request by thebithead**
Continuation of general code cleanup started by Martin in the private version of this Git repo.

I have moved some CRs around so you may need to adjust your local scripts accordingly.

### CODE REVIEW

File: pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go [hunk 1]
```diff
@@ -460,124 +459,9 @@ func createECKAllInOne(thisClient client.Client, thisScheme *runtime.Scheme, ins
 }
 
 // Delete Elastic Cloud on K8s
```
Comment (vegichan) [inline pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go]: this comment seems to be obsolete now

File: pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go [hunk 2]
```diff
@@ -423,7 +422,7 @@ func createECKAllInOne(thisClient client.Client, thisScheme *runtime.Scheme, ins
 		"ServiceAccount",
 		"StatefulSet" }
 
-	// First check that processKindOrder has all of the runtiumeObjects kinds.
+	// First check that processKindOrder has all of the runtiume Objects kinds.
```
Comment (vegichan) [inline pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go]: typo

File: pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go (hunk 1 above)
Comment (thebithead) [reply pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go]: Fixed,

File: pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go (hunk 2 above)
Comment (thebithead) [reply pkg/controller/nuodbinsightsserver/nuodbinsightsserver_ctl_internal.go]: Fixed.

**Pull request merged**